apps/dashboard/src/components/variable/utils/digest-variables.tsx

```diff
@@ -0,0 +1,60 @@
+import { createRoot } from 'react-dom/client';
+import { LiquidVariable } from '../../../utils/parseStepVariables';
+import { DigestCountSummaryPreview } from '@/components/variable/components/digest-count-summary-preview';
+import { DigestSentenceSummaryPreview } from '@/components/variable/components/digest-sentence-summary-preview';
+
+enum DIGEST_VARIABLES_ENUM {
+  COUNT_SUMMARY = 'step.digest.countSummary',
+  SENTENCE_SUMMARY = 'step.digest.sentenceSummary',
+}
+
+export const DIGEST_VARIABLES: LiquidVariable[] = [
+  {
+    label: DIGEST_VARIABLES_ENUM.COUNT_SUMMARY,
+    type: 'digest',
+    boost: 99,
+    info: () => {
```

Comment:
```suggestion
    preview: () => {
```

Reply:
This is used for codemirror only https://codemirror.net/docs/ref/#autocomplete.Completion.info